diff --git a/node_modules/@roamhq/wrtc/lib/binding.js b/node_modules/@roamhq/wrtc/lib/binding.js
index 095e3ba..09ac221 100644
--- a/node_modules/@roamhq/wrtc/lib/binding.js
+++ b/node_modules/@roamhq/wrtc/lib/binding.js
@@ -1,6 +1,7 @@
 'use strict';
 
 const os = require('os');
+
 const paths_to_try = [
   '../build/wrtc.node',
   '../build/Debug/wrtc.node',
@@ -8,14 +9,17 @@ const paths_to_try = [
   `@roamhq/wrtc-${os.platform()}-${os.arch()}`,
 ];
 
+function dynamicRequire(modulePath) {
+  return  eval(`require("${modulePath}")`);
+}
+
 let succeeded = false;
-for (const path of paths_to_try) {
+for (const packagePath of paths_to_try) {
   try {
-    module.exports = require(path);
+    module.exports = dynamicRequire(packagePath);
     succeeded = true;
     break;
   } catch (error) {
-    ;
   }
 }
 
